components/automate-chef-io/content/docs/event-feed.md

```diff
@@ -12,6 +12,8 @@ toc = true
 Use the event feed for actionable insights and operational visibility.
 The guitar strings and improved query language help you drill into infrastructure & compliance automation events quickly to isolate errors.
 
+Permission for the `event:*` action is required to interact with the **Event Feed** page. In order to search on this page the `infra:nodes:list` action is also required.
```

Comment:
@msorens @susanev Is the `event:*` action required to _interact_ or to _view_ tthe Event Feed page?

Reply:
yes

Reply:
So then, the answer would be both "view and interact."

Reply:
i guess. i think this is a little different than editor or viewer though, i dont know if that matters. there is nothing to edit on the event feed page.